com.unity.mobile.notifications/Documentation~/Unified.md

```diff
@@ -0,0 +1,174 @@
+
+The following code example demonstrates how to cancel individual notification using the ID: 
+
+```c#
+NotificationCenter.CancelScheduledNotification(id);
+NotificationCenter.CancelDeliveredNotification(id);
+```
+
+To update an existing notification, schedule a notification with the same ID. This action replaces the previously scheduled notification.
+
+## Receive notifications
+
+
+If you want to check whether your application is launched by tapping a notification, you can query the last responded notification using the following code example:
+
+```c#
+var notification = NotificationCenter.LastRespondedNotification;
+if (notification.HasValue)
+{
+    var lastNotification = notification.Value;
+}
+```
+
+This also works if you send notification to the background and tap the notification to bring it back to foreground. You can query the properties of returned notification.
+
+If you want to receive notifications that are displayed while your application is in the foreground (by default such notifications aren't presented to the user), you must subscribe to an event, as demonstrated in the following code example:
+
+```c#
+void Start()
+{
+    NotificationCenter.OnNotificationReceived += NotificationReceived;
+}
+
+void NotificationReceived(Notification notification)
+{
+    Debug.Log($"Received notification with title: {notification.Title}");
+}
+```
+
+## Advanced features
+
+You can manage notifications with the advanced features such as categories and groups. You can also present notifications by setting up badges, alerts, and device sound.
+### Categories and groups
+
+The terms Category and Group indicate different things on Android and iOS.
+
+**Terminology differences**:
+
+* **Category** - a channel on Android, a category on iOS.
+* **Group** - a group on Android, a thread on iOS.
+
+By default, notifications have no category on iOS, while on Android they are sent to the default channel that you set when initializing the system. There is a separate overload of `ScheduleNotification` taking category as a parameter, that sends notification to the provided category or channel respectively. To create the [category](../api/Unity.Notifications.iOS.iOSNotificationCenter.html#Unity_Notifications_iOS_iOSNotificationCenter_SetNotificationCategories_System_Collections_Generic_IEnumerable_Unity_Notifications_iOS_iOSNotificationCategory__) or [channel](../api/Unity.Notifications.Android.AndroidNotificationCenter.html#Unity_Notifications_Android_AndroidNotificationCenter_RegisterNotificationChannel_Unity_Notifications_Android_AndroidNotificationChannel_), you must use platform specific APIs.
```

Comment:
Although, I could build the docs locally and these links worked, I'm not sure if such kind of cross references would work after the merge. Let me come back on these links.

Reply:
I have found the guidance on cross referencing package API docs. Although these links work locally, they may or may not work in the end. As per the guidance, I'm adding cross references using uid for the methods.